apps/ledger-live-mobile/e2e/utils/cliUtils.ts

```diff
@@ -0,0 +1,131 @@
+import customLockScreenRemove from "@ledgerhq/live-cli/src/commands/device/customLockScreenRemove";
+import devDeviceAppsScenario from "@ledgerhq/live-cli/src/commands/device/devDeviceAppsScenario";
+import deviceAppVersion from "@ledgerhq/live-cli/src/commands/device/deviceAppVersion";
+import deviceInfo from "@ledgerhq/live-cli/src/commands/device/deviceInfo";
+import deviceSDKFirmwareUpdate from "@ledgerhq/live-cli/src/commands/device/deviceSDKFirmwareUpdate";
+import deviceSDKGetBatteryStatuses from "@ledgerhq/live-cli/src/commands/device/deviceSDKGetBatteryStatuses";
+import deviceSDKGetDeviceInfo from "@ledgerhq/live-cli/src/commands/device/deviceSDKGetDeviceInfo";
+import deviceSDKToggleOnboardingEarlyCheck from "@ledgerhq/live-cli/src/commands/device/deviceSDKToggleOnboardingEarlyCheck";
+import deviceVersion from "@ledgerhq/live-cli/src/commands/device/deviceVersion";
+import discoverDevices from "@ledgerhq/live-cli/src/commands/device/discoverDevices";
+import firmwareRepair from "@ledgerhq/live-cli/src/commands/device/firmwareRepair";
+import firmwareUpdate from "@ledgerhq/live-cli/src/commands/device/firmwareUpdate";
+import genuineCheck from "@ledgerhq/live-cli/src/commands/device/genuineCheck";
+import getBatteryStatus from "@ledgerhq/live-cli/src/commands/device/getBatteryStatus";
+import getDeviceRunningMode from "@ledgerhq/live-cli/src/commands/device/getDeviceRunningMode";
+import i18n from "@ledgerhq/live-cli/src/commands/device/i18n";
+import listApps from "@ledgerhq/live-cli/src/commands/device/listApps";
+import managerListApps from "@ledgerhq/live-cli/src/commands/device/managerListApps";
+import proxy from "@ledgerhq/live-cli/src/commands/device/proxy";
+import reinstallConfigurationConsent from "@ledgerhq/live-cli/src/commands/device/reinstallConfigurationConsent";
+import repl from "@ledgerhq/live-cli/src/commands/device/repl";
+import speculosList from "@ledgerhq/live-cli/src/commands/device/speculosList";
+import balanceHistory from "@ledgerhq/live-cli/src/commands/live/balanceHistory";
+import countervalues from "@ledgerhq/live-cli/src/commands/live/countervalues";
+import envs from "@ledgerhq/live-cli/src/commands/live/envs";
+import exportAccounts from "@ledgerhq/live-cli/src/commands/live/exportAccounts";
+import ledgerKeyRingProtocol from "@ledgerhq/live-cli/src/commands/live/ledgerKeyRingProtocol";
+import ledgerSync from "@ledgerhq/live-cli/src/commands/live/ledgerSync";
+import liveData from "@ledgerhq/live-cli/src/commands/live/liveData";
+import portfolio from "@ledgerhq/live-cli/src/commands/live/portfolio";
+import synchronousOnboarding from "@ledgerhq/live-cli/src/commands/live/synchronousOnboarding";
+import user from "@ledgerhq/live-cli/src/commands/live/user";
+import version from "@ledgerhq/live-cli/src/commands/live/version";
+import swap from "@ledgerhq/live-cli/src/commands/ptx/swap";
+
+export const CLI = {
```

Comment:
Can we move the cliUtils present in LLD to Ledger-live-common and use it in LLM as well?

Reply:
It create cyclic dependancies between ledger-live and cli modules, so we can't :/